File: wlr/OGUR/OGUR/GameObjects/Elements.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework;

namespace OGUR.GameObjects
{
    public class Elements
    {
        public const int NORMAL = 0;
        public const int FIRE = 1;
        public const int WATER = 2;
        public const int EARTH = 3;
        public const int AIR = 4;
        public const int LIGHT = 5;
        public const int DARK = 6;
        public const int PHYSICAL = 7;
        public const int MENTAL = 8;

        public static readonly int[] Values = 
        {
            NORMAL,
            FIRE,
            WATER,
            EARTH,
            AIR,
            LIGHT,
            DARK,
            PHYSICAL,
            MENTAL
        };

        public static readonly Color[] Colors =
        {
            Color.Gray,
            Color.Black,
            Color.White,
            Color.Red,
            Color.Green,
            Color.Blue,
            Color.Purple,
            Color.Yellow,
            Color.Orange
        };

    }
}

File: aigilas/src/aigilas/entities/Elements.java
package aigilas.entities;

import com.badlogic.gdx.graphics.Color;

public enum Elements {
    NORMAL(0, Color.GRAY),
    FIRE(1, Color.RED),
    WATER(2, Color.BLUE),
    EARTH(3, Color.GREEN),
    AIR(4, Color.YELLOW),
    LIGHT(5, Color.WHITE),
    DARK(6, Color.BLACK),
    PHYSICAL(7, Color.ORANGE),
    MENTAL(8, Color.MAGENTA);

    public final Color Tint;
    public final int Value;

    private Elements(int value, Color color) {
        Value = value;
        Tint = color;
    }

    public static Elements get(String element) {
        for(Elements e:values()){
            if(e.toString().equalsIgnoreCase(element)){
                return e;
            }
        }
        return null;
    }
}
